### ISSUE TITLE
Android Talk app states that app is not installed on server even it is

### ISSUE THREAD

**Issue by MrColumbo** [nextcloud/talk-android#1735]
## Steps to reproduce
1. Open Android App and add server url
2. Get Screen to gran access
3. after that error pops up and initial screen is shown

### Expected behaviour
talk on android should be usable

### Actual behaviour
Tell us what happens instead
step 3

## Device information
OnePlus 6 t

**Android version:** 10/…
11

**Talk version:** 11.0.0
on android version from nov 29th 2021
## Server information

**Nextcloud version:** (see admin overview page: `/index.php/settings/admin/overview`)
23

**Talk version:** (see apps admin page: `/index.php/settings/apps`)
13
**Custom Signaling server configured:** yes/no and version (see talk admin settings: `/index.php/index.php/settings/admin/talk#signaling_server`)
no

**Custom TURN server configured:** yes/no (see talk admin settings: `/index.php/settings/admin/talk#turn_server`)
no

**Custom STUN server configured:** yes/no (see talk admin settings: `/index.php/settings/admin/talk#stun_server`)
no

### Server log (data/nextcloud.log)
<details>

```


92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /index.php/login/flow HTTP/1.1" 200 8405 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /core/css/login/authpicker.css?v=cbee0c0d-0 HTTP/1.1" 200 691 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /apps/files_rightclick/css/app.css?v=62abc69f-0 HTTP/1.1" 200 745 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /core/css/guest.css?v=cbee0c0d-0 HTTP/1.1" 200 6154 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /index.php/apps/theming/styles?v=0 HTTP/1.1" 200 1476 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /core/js/dist/files_fileinfo.js?v=cbee0c0d-0 HTTP/1.1" 200 10021 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /core/js/dist/files_client.js?v=cbee0c0d-0 HTTP/1.1" 200 43456 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /apps/files_sharing/l10n/de_DE.js?v=cbee0c0d-0 HTTP/1.1" 200 6157 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /apps/files_sharing/js/dist/main.js?v=cbee0c0d-0 HTTP/1.1" 200 1634 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /apps/accessibility/l10n/de_DE.js?v=cbee0c0d-0 HTTP/1.1" 200 1815 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /core/l10n/de_DE.js?v=cbee0c0d-0 HTTP/1.1" 200 22770 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /apps/accessibility/js/accessibilityoca.js?v=cbee0c0d-0 HTTP/1.1" 200 6147 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /apps/files_rightclick/l10n/de_DE.js?v=cbee0c0d-0 HTTP/1.1" 200 1053 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /apps/files_rightclick/js/script.js?v=cbee0c0d-0 HTTP/1.1" 200 3781 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /apps/files_videoplayer/js/files_videoplayer-main.js?v=cbee0c0d-0 HTTP/1.1" 200 20034 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /core/js/login/authpicker.js?v=cbee0c0d-0 HTTP/1.1" 200 728 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /apps/files_rightclick/js/files.js?v=cbee0c0d-0 HTTP/1.1" 200 1832 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /core/img/background.png?v=0 HTTP/1.1" 200 10118 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /apps/theming/l10n/de_DE.js?v=cbee0c0d-0 HTTP/1.1" 200 2151 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /core/img/actions/confirm-white.svg?v=2 HTTP/1.1" 200 904 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /index.php/core/js/oc.js?v=cbee0c0d HTTP/1.1" 200 2483 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /core/js/dist/main.js?v=cbee0c0d-0 HTTP/1.1" 200 487531 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /index.php/js/core/merged-template-prepend.js?v=cbee0c0d-0 HTTP/1.1" 200 3955 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /index.php/apps/accessibility/css/user-a82fd95db10ff25dfad39f07372ebe37 HTTP/1.1" 200 5283 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:47 +0000] "GET /index.php/svg/core/logo/logo?color=ffffff&v=1 HTTP/1.1" 200 1583 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:58 +0000] "GET /index.php/login/flow/grant?clientIdentifier=&stateToken=0vEOdeCmI6Q9XXX...HTTP/1.1" 303 894 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:58 +0000] "GET /index.php/login?redirect_url=/index.php/login/flow/grant?clientIdentifier%3D%26stateToken%3D0vEOdeCmXXXX..... HTTP/1.1" 200 7328 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:58 +0000] "GET /core/js/dist/login.js?v=cbee0c0d-0 HTTP/1.1" 200 271092 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:59 +0000] "GET /core/img/actions/toggle.svg HTTP/1.1" 200 807 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:24:59 +0000] "GET /core/img/loading-dark.gif HTTP/1.1" 200 5180 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:25:09 +0000] "POST /index.php/login HTTP/1.1" 303 1008 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

127.0.0.1 - - [03/Dec/2021:18:25:15 +0000] "OPTIONS * HTTP/1.0" 200 126 "-" "Apache/2.4.51 (Debian) PHP/8.0.13 (internal dummy connection)"

92.116.123.123- - [03/Dec/2021:18:25:15 +0000] "GET /index.php/login/challenge/totp?redirect_url=/index.php/login/flow/grant?clientIdentifier%3D%26stateToken%3D0vEOdeCmXXXX..... HTTP/1.1" 200 7475 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:25:15 +0000] "GET /custom_apps/twofactor_totp/css/style.css?v=54d09f91-0 HTTP/1.1" 200 666 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:25:15 +0000] "GET /apps/accessibility/l10n/de.js?v=cbee0c0d-0 HTTP/1.1" 200 1810 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:25:15 +0000] "GET /apps/files_rightclick/l10n/de.js?v=cbee0c0d-0 HTTP/1.1" 200 1031 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:25:15 +0000] "GET /apps/theming/l10n/de.js?v=cbee0c0d-0 HTTP/1.1" 200 2137 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:25:15 +0000] "GET /apps/files_sharing/l10n/de.js?v=cbee0c0d-0 HTTP/1.1" 200 6167 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:25:15 +0000] "GET /custom_apps/twofactor_totp/img/app.svg HTTP/1.1" 200 1542 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:25:15 +0000] "GET /core/l10n/de.js?v=cbee0c0d-0 HTTP/1.1" 200 22150 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:25:31 +0000] "POST /index.php/login/challenge/totp?redirect_url=/index.php/login/flow/grant?clientIdentifier%3D%26stateToken%3D0vEOdeCmXXXX..... HTTP/1.1" 303 1209 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:25:31 +0000] "GET /index.php/login/flow/grant?clientIdentifier=&stateToken=0vEOdeCmI6Q9XXX...HTTP/1.1" 200 7342 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:25:31 +0000] "GET /core/js/login/grant.js?v=cbee0c0d-0 HTTP/1.1" 200 819 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"

92.116.123.123- - [03/Dec/2021:18:25:37 +0000] "POST /index.php/login/flow HTTP/1.1" 303 880 "-" "Oneplus ONEPLUS A6003 (Nextcloud Talk)"


```
</details>

**Comment by mahibi** [comment]
Dear @MrColumbo Please update to the latest Android Talk App.
Android Talk 11.0 is not able to handle the newest Talk 13 on Server.
closing as duplicate of https://github.com/nextcloud/talk-android/issues/1663

**Comment by MrColumbo** [comment]
Dear @mahibi thanks for looking into the issue. Unfortunately I made a mistake when I opened the issue. I am actually using Talk 13.0 on my android device

**Comment by AndyScherzinger** [comment]
@mahibi I took the liberty to re-open. So what you are say @MrColumbo is that (now) you are using v13 on the server _and_ the client, right? Using client "older" then the server version is not supported and may or may not lead to issues (usually does). Since the client is backward compatible but not forward compatible.

**Comment by MrColumbo** [comment]
I just double checked .....both clients and server are 13.0.0 - thanks for helping!

**Comment by MrColumbo** [comment]
When I reported the issue I already was on the latest version. It was just a typo when I stayed the Android version and the version date I initially mentioned (Nov 29th 2021) is the 13.0 version. 
The issue still exists

**Comment by mahibi** [comment]
Is it possible for you to try with other devices? Do all of them have this problem?

Could you temporarily create a test-account for me on your nextcloud instance so i can have a look? If yes, please send it to marcel.hibbe@nextcloud.com

**Comment by MrColumbo** [comment]
I just tried with the smartphone from my wife and it worked. I never thought that this could be a device issue .....

**Comment by mahibi** [comment]
ok that's interesting though it makes debugging a bit trickier..

- which OS of Android do you use? Is it the plain "OxygenOS" that was shipped with your OnePlus 6 t or did you install some other OS like LineageOS?
- from where did you install the Talk App? Playstore or Fdroid?

**Comment by mahibi** [comment]
if you are willing to install Android Studio to see what is logged when you log in, that would be helpful.
You would need to install Android Studio, connect your smartphone to Android Studio via USB debugging and then watch the output from "Logcat" while trying to log in to your Nextcloud instance.
Just let me know if you are willing to do this or if you need help..

**Comment by MrColumbo** [comment]
today I though I just delete the app cache and data on my device and try it again but that did not help ......
How can one device work fine while the other does not?

**Comment by mahibi** [comment]
> How can one device work fine while the other does not?

i have no idea so far, sometimes there can be weird effects with different vendors & android versions. Thus the questions from my last 2 comments..

**Comment by MrColumbo** [comment]
I fixed it. Beside different smartphones I am using 2FA. I was able to login without an error by using an app token password.
Before I played around by using the version from fdroid  - there was no difference. Sometimes I got a dashboard screen like the talk app is a browser and showed the mobile version.
Anyway .....it works now. Sorry .....i did not notice your request for a test account .....was quite busy last week .....

thanks for your support
